src/Sarif.Viewer.VisualStudio/ErrorList/SarifSnapshot.cs

```diff
@@ -110,13 +85,13 @@ public override bool TryGetValue(int index, string columnName, out object conten
                 }
                 else if (columnName == StandardTableKeyNames.Text)
                 {
-                    content = _errors[index].ShortMessage;
+                    content = SdkUiUtilities.UnescapeBrackets(_errors[index].ShortMessage);
                 }
                 else if (columnName == StandardTableKeyNames.FullText)
                 {
                     if (!string.IsNullOrEmpty(_errors[index].Message) && _errors[index].Message.Trim() != _errors[index].ShortMessage.Trim())
                     {
-                        content = _errors[index].Message;
+                        content = SdkUiUtilities.UnescapeBrackets(_errors[index].Message);
```

Comment:
>SdkUiUtilities [](start = 34, length = 14)

NIT casing here should be SdkUIUtilities (see UI vs Ui)

Reply:
Fixed. SdkUIUtilitiesTests too